tests/runtests.sh

```diff
@@ -90,7 +95,7 @@ function fail () {
 }
 
 function start_toaster() {
-    mkdir $tempdir/toasterbuild
+    mkdir -p $tempdir/$toasterbuild
```

Comment:
it should not be there anymore. It was part of my debugging the flow and I
meant to rm it.
-b

On Sun, Aug 21, 2016 at 5:21 PM, Randy Witt <notifications@github.com>
wrote:

> In tests/runtests.sh
> <https://github.com/crops/toaster-container/pull/8#discussion_r75606836>:
>
> > @@ -90,7 +95,7 @@ function fail () {
> >  }
> >
> >  function start_toaster() {
> > -    mkdir $tempdir/toasterbuild
> > +    mkdir -p $tempdir/$toasterbuild
>
> Why is the -p added? It was partly intended to make sure it wasn't trying
> to create something off in the weeds where not expected.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/crops/toaster-container/pull/8/files/9dda39d3cf58b81c6820ef1528d2728a962f26a1..ce007d22f3af1722580467f78fb7d1d616c6e320#r75606836>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACkxjcRf0h0DyziG5fLVPsAxE3ojduCZks5qiOtzgaJpZM4Jo75K>
> .
>